cyber-security/modules/csls_s3_object_logging/cloudwatch.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_cloudwatch_event_rule" "s3_events" {
+  name        = local.cloudwatch_event_rule_name
+  description = "S3 object level logging rule"
+
+  tags = merge(var.tags, map("Name", local.cloudwatch_event_rule_name))
+
+  event_pattern = <<PATTERN
```

Comment:
Not sure if you can use a data template in JSON, Alex prefers this to heredoc

Reply:
Done